tests/topotests/lib/packet/igmp/igmp.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+#
+#  SPDX-License-Identifier: BSD-2-Clause
+#
+#  igmp.py
+#  Part of NetDEF CI System
+#
+#  Copyright (c) 2025 by
+#  Network Device Education Foundation, Inc. ("NetDEF")
+#
+import struct
+
+from scapy.all import Packet
+from scapy.layers.inet import IP, IPOption_Router_Alert
+from scapy.layers.l2 import Ether
+from scapy.packet import bind_layers
+from scapy.sendrecv import sendp
+
+def calculate_checksum(packet):
+    if len(packet) % 2 == 1:
+        packet += b'\0'
+    s = sum(struct.unpack("!%dH" % (len(packet) // 2), packet))
+    s = (s >> 16) + (s & 0xffff)
+    s += s >> 16
+    return ~s & 0xffff
```

Comment:
This function was also added to topotests/lib/packet/helper.py. did you intended to copy one copy only?